supergloo/docs/mesh/install.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "Installing a Mesh"
+weight: 2
+---
+
+# Overview
+
+SuperGloo can be used to install, upgrade, and uninstall a supported mesh.
+
+Currently supported meshes for installation:
+
+- Istio
+
+## Installing Istio with SuperGloo
+
+First, ensure that SuperGloo has been initialized in your kubernetes cluster via `supergloo init` or the 
+[Supergloo Helm Chart](https://github.com/solo-io/supergloo/tree/master/install/helm/supergloo). See the [installation
+instructions](install) for detailed instructions on installing SuperGloo.
+
+Once SuperGloo has been installed, we'll create an Install CRD with configuration parameters which will then
+trigger SuperGloo to begin the mesh installation.
+
+This can be done in one of two ways:
+
+#### Option 1: Using the `supergloo` CLI:
+
+```bash
+supergloo install istio --name istio --installation-namespace istio-system --mtls=true --auto-inject=true
+```
+
+See `supergloo install istio --help` for the full list of installation options for istio.
+
+
+#### Option 2: Using `kubectl apply` on a yaml file:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: supergloo.solo.io/v1
+kind: Install
+metadata:
+name: my-istio
+spec:
+istio:
+  enableAutoInject: true
+  enableMtls: true
+  installGrafana: true
+  installJaeger: true
+  installPrometheus: true
+  installationNamespace: istio-system
+  istioVersion: 1.0.6  
+EOF
+```
+
+Once you've created the Install CRD, you can track the progress of the Istio installation:
+
+```bash
+kubectl get pod -n istio-system --watch
+
+NAME                                      READY     STATUS              RESTARTS   AGE
+grafana-7f6cd4bf56-xst2n                  1/1       Running             0          27s
+istio-citadel-796c94878b-59gw6            1/1       Running             0          26s
+istio-cleanup-secrets-xd6f8               0/1       Completed           0          27s
+istio-galley-6c68c5dbcf-z6j5k             1/1       Running             0          27s
+istio-pilot-c5dddb4b9-nb6fd               2/2       Running             0          26s
+istio-policy-977d74ff4-669vk              2/2       Running             0          27s
+istio-sidecar-injector-6d8f88c98f-5f58t   1/1       Running             0          26s
+istio-telemetry-5f79796bf6-fl4sn          2/2       Running             0          27s
+istio-tracing-7596597bd7-lc92t            1/1       Running             0          26s
+prometheus-76db5fddd5-55r6d               1/1       Running             0          26s
+
+````
```

Comment:
Think you got one too many single quotes here. At the moment, it renders like this:

<img width="942" alt="image" src="https://user-images.githubusercontent.com/53201/54033440-8cbfca00-4171-11e9-93ed-8aab03579c3f.png">